docs/kubernetes/user-guides/basic.md

```diff
@@ -0,0 +1,83 @@
+# Kubernetes Basic User Guide
+This guide gives a simple example on how to provision zonal and regional PDs in single-zone and regional clusters.
+
+**Note:** Regional cluster support only available in beta starting with
```

Comment:
maybe move/duplicate this note in the "regional PD" section

Reply:
I duplicated it because you can use single-zone pds in a regional cluster too